nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -272,6 +272,28 @@ in
         '';
       };
 
+      logLevel = mkOption {
+        type = types.str;
+        default = "VERBOSE";
+        description = ''
+          Gives the verbosity level that is used when logging messages from sshd(8). The possible values are:
+          QUIET, FATAL, ERROR, INFO, VERBOSE, DEBUG, DEBUG1, DEBUG2, and DEBUG3. The default is INFO. DEBUG and DEBUG1
+          are equivalent. DEBUG2 and DEBUG3 each specify higher levels of debugging output. Logging with a DEBUG level
+          violates the privacy of users and is not recommended.
+        '';
+      };
+
+      useDns = mkOption {
+        type = types.bool;
+        default = true;
```

Comment:
current default [is](https://www.ietf.org/mail-archive/web/dnsop/current/msg13358.html) ```UseDNS no```
silently changing it to ```yes``` on nixpkgs update might break someone's setup up to inability to login